TreeDB/collections/command_wal_test.go

```
@@ -828,6 +828,787 @@ func TestCollectionCommandWALUpdateBSONSetUniqueIndexChangePublishesAppliedLSN(t
+	time.Sleep(25 * time.Millisecond)
+	select {
+	case err := <-drainDone:
+		col.writeDomain.mu.Unlock()
+		t.Fatalf("owner drain completed while owner domain lock was held: %v", err)
+	default:
+	}
+
+	coordAvailable := make(chan struct{})
+	go func() {
+		coord.mu.Lock()
+		coord.mu.Unlock()
```

golangci-lint (2.12.2)

[error] SA2001: empty critical section

(staticcheck)